packages/modelviewer.dev/package.json

```diff
@@ -11,7 +11,14 @@
     "update:package-lock": "rm ./package-lock.json && npm i --only=production",
     "create-legacy-bundles": "node ./scripts/create-legacy-bundles.js"
   },
-  "author": "DevXR Contributors",
+  "contributors": [
+    "Jordan Santell <jsantell@google.com>",
+    "Chris Joel <cdata@google.com>",
+    "Emmett Lalish <elalish@google.com",
+    "Ricardo Cabello <ricardocabello@google.com>",
+    "Matt Small <mbsmall@google.com>",
+    "Yuin Chin <yuin@google.com>"
```

Comment:
Yuin Chien

Reply:
Thanks! That typo's been there a long time...